catalog/core/catalog-core-api/src/main/java/ddf/catalog/data/types/Media.java

```diff
@@ -96,4 +96,11 @@
      */
     String WIDTH = "media.width-pixels";
 
+    /**
+     * {@link ddf.catalog.data.Attribute} name for accessing the duration in seconds of a {@link ddf.catalog.resource.Resource}
+     * of a {@link Metacard}. <br/>
+     */
+    String DURATION = "media.duration";
```

Comment:
I am not seeing media.duration being used anywhere.  Does tika expose duration in the metadata?